src/MCPClient/tests/test_sanitize.py

```diff
@@ -342,7 +320,6 @@ def test_sanitize_name_raises_valueerror_on_empty_string():
         sanitize_names.sanitize_name("")
 
 
-@pytest.mark.xfail(reason="This is affected by the microservice_agents.json fixture")
```

Comment:
CC @replaceafill - hopefully not needed anymore. I'll be merging soon, as soon as I get the approval. We asked Evelyn to take a look at the METS.